vss-testclient/testclient.py

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/python3
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+
+import argparse, json, sys
+from cmd2 import Cmd, with_argparser, with_category
+import ipaddress, queue, time
+from clientComm import VSSClientComm
+
+ap_getServerAddr = argparse.ArgumentParser()
+ap_connect = argparse.ArgumentParser()
+ap_connect.add_argument('-i', "--insecure", default=False, action="store_true", help='Connect in insecure mode')
+ap_disconnect = argparse.ArgumentParser()
+ap_authorize = argparse.ArgumentParser()
+ap_authorize.add_argument('Token', help='JWT for authorizing the client.')
+ap_setServerAddr = argparse.ArgumentParser()
+ap_setServerAddr.add_argument('IP', help='VSS Server IP Address')
+ap_setServerAddr.add_argument('Port', type=int, help='VSS Server Websocket Port')
+ap_setValue = argparse.ArgumentParser()
+ap_setValue.add_argument("Parameter", help="Parameter to be set")
+ap_setValue.add_argument("Value", help="Value to be set")
+ap_getValue = argparse.ArgumentParser()
```

Comment:
Not sure this code is related :-), but for the usability: Wouldn't it be better to just use "get" and "set" insted of "getValue" and "setValue" (as those two are probably used most often)

Reply:
_get_ and _set_ are both keywords in the Cmd2 library to work with internal debug variables.
We can ofcourse override them for the testclient, but then this will be counter-productive for further development, in my opinion. 
Hence I suggest to not modify the code for this purpose.

Reply:
makes sense 👍